website/docs/reference/warehouse-setups/infer-setup.md

```diff
@@ -0,0 +1,125 @@
+
+
+<h2> Installing {frontMatter.meta.pypi_package} </h2>
+
+pip is the easiest way to install the adapter:
+
+<code>pip install {frontMatter.meta.pypi_package}</code>
+
+<p>Installing <code>{frontMatter.meta.pypi_package}</code> will also install <code>dbt-core</code> and any other dependencies.</p>
+
+<h2> Configuring {frontMatter.meta.pypi_package} </h2>
+
+<p>For further info, refer to the GitHub repository: <a href={`https://github.com/${frontMatter.meta.github_repo}`}>{frontMatter.meta.github_repo}</a></p>
+
+
+## Connecting to Infer with **dbt-infer**
+
+Infer allows you to perform advanced ML Analytics within SQL as if native to your data warehouse.
+To do this Infer uses a variant called <a href="https://docs.getinfer.io/docs/reference">SQL-inf</a> 
+which defines as set of primitive ML commands from which 
+you can build advanced analysis for any business use case.
+Learn more about SQL-inf and Infer <a href="https://docs.getinfer.io/">here</a>.
+
+The `dbt-infer` package allow you to use SQL-inf easily within your DBT models. 
+You can read more about the `dbt-infer` package <a href="https://dbt.getinfer.io/">here</a>.
+
+Before using SQL-inf in your DBT models you need to setup an Infer account and generate an API-key for the connection.
+You can read how to do that <a href="https://dbt.getinfer.io/docs/getting_started#sign-up-to-infer">here</a>.
+
+The profile configuration in `profiles.yml` for `dbt-infer` should look something like this:
+
+<File name='~/.dbt/profiles.yml'>
+
+```yaml
+<profile-name>:
+  target: <target-name>
+  outputs:
+    <target-name>:
+      type: infer
+      url: "<infer-api-endpoint>"
+      username: "<infer-api-username>"
+      apikey: "<infer-apikey>"
+      data_config:
+        [configuration for your underlying data warehouse]  
+```
+
+</File>
+
+Note that you need to also have installed the adapter package for your underlying data warehouse.
+For example, if your data warehouse is BigQuery then you need to also have installed the appropriate `dbt-bigquery` package.
+
+### Description of Infer Profile Fields
+
+| Field      | Required | Description                                                                                             |
+|------------|----------|---------------------------------------------------------------------------------------------------------|
+| `type`     | Yes | Must be set to `infer`. This must be included either in `profiles.yml` or in the `dbt_project.yml` file. |
+| `url`      | Yes | The host name of the Infer server to connect to. Typically this is `https://app.getinfer.io`.     |
+| `username` | Yes | Your Infer username - the one you use to login.                                                         |
+| `apikey`   | Yes | Your Infer api key.                                                                                     |
+| `data_config` | Yes | The configuration for your underlying data warehouse.                                                   |
```

Comment:
I think an example of this would be very helpful. how should it be structured?

Reply:
Added a bit more info and an example